src/applications/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -392,92 +390,72 @@ const countries = [
   'Zimbabwe'
 ];
 
-export const militaryPostOfficeTypeLabels = { // TODO: determine whether these are necessary
-  APO: 'Army Post Office',
-  FPO: 'Fleet Post Office',
-  DPO: 'Diplomatic Post Office'
-};
-
 const addressUISchema = (addressName, title) => {
   return {
+    'ui:order': [
+      'country',
+      'addressLine1',
+      'addressLine2',
+      'addressLine3',
+      'city',
+      'state',
+      'zipCode'
+    ],
     'ui:title': title,
-    type: {
-      'ui:title': 'Type',
-      'ui:options': {
-        labels: typeLabels
-      }
-    },
     country: {
       'ui:title': 'Country'
     },
-    state: {
-      'ui:title': 'State',
-      'ui:options': {
-        labels: stateLabels,
-        hideIf: formData => {
-          return (
-            _.get(formData, `veteran[${addressName}].country`) !== 'USA'
-          );
-        }
-      }
-    },
     addressLine1: {
-      'ui:title': 'Street'
+      'ui:title': 'Street address'
     },
     addressLine2: {
-      'ui:title': 'Line 2'
+      'ui:title': 'Street address (optional)'
```

Comment:
I stole this text from Letters, since that's our only PCIU precedent. I'm intending to have Peggy weigh in on content (along with validation content) in another PR - this is mostly just to make things work.